enrichment-definition/src/main/java/eu/europeana/enrichment/model/impl/KeywordNamedEntity.java

```diff
@@ -0,0 +1,126 @@
+package eu.europeana.enrichment.model.impl;
+
+import java.util.List;
+import java.util.Map;
+
+import org.bson.types.ObjectId;
+
+import dev.morphia.annotations.Entity;
+import dev.morphia.annotations.Id;
+
+@Entity(value="KeywordNamedEntity")
+public class KeywordNamedEntity extends NamedEntityImpl{
```

Comment:
I assume this is the deprecated class that we want to remove, please mark it as deprecated for the time being